test/built-ins/IteratorPrototype/Symbol.iterator/return-val.js

```diff
@@ -1,19 +1,32 @@
 // Copyright (C) 2015 the V8 project authors. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 25.1.2.1
-description: Descriptor for `name` property
+esid: sec-%iteratorprototype%-@@iterator
```

Comment:
Is this the correct way of going about this? I copied directly from the `AsyncIterator` test above, did I miss something?

Reply:
The `esid` is supposed to be the same as the HTML anchor of the relevant section of the spec. [(docs)](https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#esid) In this case, it's https://tc39.es/ecma262/#sec-%iteratorprototype%-@@iterator, so this value looks correct.